AppKit/CoreAnimation/CPAnimationContext.j

```diff
@@ -162,14 +162,19 @@ CPLog.debug(_cmd + "context stack =" + _CPAnimationContextStack);
     var completionFunction = function()
     {
         if (needsFrameTimer)
+        {
             [self stopFrameUpdaterWithIdentifier:objectId];
+            [anObject _setForceDOMUpdates:YES];
```

Comment:
This should be set and unset in each of the view animator proxy methods using the convenience method:
```[self _setTargetValue:aTargetValue withKeyPath:aKeyPath fallback:handler completion:handler];```
Be aware that the completion handler needs this flag to be on, but not the fallback handler (the handler being executed when animations are unavailable).